.github/actions/pr_notifier/pr_notifier.py

```diff
@@ -69,6 +69,13 @@ def is_waiting(labels):
     return False
 
 
+def is_contrib(labels):
+    for label in labels:
```

Comment:
small nit - these few lines can be reduced to `return any(label.name == "contrib" for label in labels)`